README.md

```diff
@@ -22,60 +18,27 @@ This theme is available in 4 variations:
 [3]: https://github.com/HerbFargus/es-theme-tronkyfran/tree/dark
 [4]: https://github.com/HerbFargus/es-theme-tronkyfran/tree/dark-optimized
 
-## Changelog
+## Install `tronkyfran`
+
+1. Close EmulationStation.
+
+2. Create folder `~/.emulationstation/themes` if not exists.
+
```

Comment:
Minor grammatical

2. Create folder `~/.emulationstation/themes` if it does not exist.